apps/naive/lib/naive/strategy.ex

```diff
@@ -407,13 +400,15 @@ defmodule Naive.Strategy do
            tick_size: tick_size,
            step_size: step_size
          } = position,
-         _settings
+         _settings,
+         trader_id
        ) do
     [price, quantity] =
       order_helper(symbol, id, "LIMIT", "SELL", price, quantity, tick_size, step_size)
 
     case @exchange_client.order_limit_sell(symbol, quantity, price) do
       {:ok, %Exchange.Order{} = order} ->
+        new_order(order, trader_id)
         {:ok, %{position | sell_order: order, position: python_position}}
```

Comment:
Does `execute_decision` belong in this file? I'm not sure if it should be the strategy's responsibility to submit an order to binance

Reply:
I think that this is the place for it, since the brunt of the strategy is now done by the `Strategies` app. I think this goes along the other comment for renaming things, maybe this file should be `trader.ex`

Reply:
+1 to move this to a separate module. I'll try to talk a little bit more about structuring during our meeting today, but this project is a salad. All mixed in. Complicated umbrella for major parts but none for control flow and separation of concerns in a way we can actually follow the path. 

The "path" the app takes is fairly linear, but the way it is right now it bounces between places erratically.

I think for the concern of this PR, I'd +1 @NeoVier. Leave strategies to decision making, creating orders on exchanges could be elsewhere. I'll go deeper into this later @MatheusBuss, give you some directions about this (finally I can help!)